test/e2e/azure_logcollector.go

```diff
@@ -237,13 +237,9 @@ func windowsK8sLogs(execToPathFn func(outputFileName string, command string, arg
 			"hyperv-operation.log",
 			"Get-WinEvent", "-LogName Microsoft-Windows-Hyper-V-Compute-Operational | Select-Object -Property TimeCreated, Id, LevelDisplayName, Message | Sort-Object TimeCreated | Format-Table -Wrap -Autosize",
 		),
-		execToPathFn(
-			"docker.log",
-			"get-eventlog", "-LogName Application -Source Docker | Select-Object Index, TimeGenerated, EntryType, Message | Sort-Object Index | Format-Table -Wrap -Autosize",
-		),
 		execToPathFn(
 			"containers.log",
-			"docker", "ps -a",
+			"ctr.exe", "-n k8s.io containers list",
```

Comment:
We should include `ctr task list` as well.  There have been times when there were containers left over but tasks where removed.

Reply:
updated